src/APIFunctions/User.js

```diff
@@ -173,7 +173,7 @@ export async function checkIfUserExists(email) {
 export function filterUsers(users, filterID) {
   let filteredUsers = users.filter((user) => {
     if (filterID === userFilterType.VALID) {
-      return (user.accessLevel >= membershipState.MEMBER);
+      return (user.accessLevel >= membershipState.ALUMNI);
```

Comment:
this implementation would be problematic if a member specific functionality uses this function

Reply:
accesslevel would be a problem because changing the numbers to role mapping would invalidate the accessLevels of existing users

Reply:
TokenFunctions.js line 10 may need updating

Reply:
an alumn after auth would only have access to their profile and signedoutroutes